roles/confluent.variables/vars/main.yml

```diff
@@ -312,6 +312,13 @@ kafka_broker_properties:
                             kafka_broker_truststore_path, kafka_broker_truststore_storepass, kafka_broker_keystore_path, kafka_broker_keystore_storepass, kafka_broker_keystore_keypass,
                             plain_jaas_config, kafka_broker_keytab_path, kafka_broker_kerberos_principal|default('kafka'), kerberos_kafka_broker_primary,
                             sasl_scram_users.admin.principal, sasl_scram_users.admin.password, rbac_enabled_public_pem_path ) }}"
+  controlplane_listener:
+    enabled: "{{ kafka_broker_configure_control_plane_listener|bool and kafka_broker_configure_multiple_listeners|bool }}"
+    properties:
+      # The property must be defined regardless if it is enabled or not.
```

Comment:
this comment doesnt match the logic in line 316. if `kafka_broker_configure_control_plane_listener = false` or `kafka_broker_configure_multiple_listeners = false`, the property `control.plane.listener.name` will not get set

Reply:
Comment is now removed because the logic was put in variable definition so that `kafka_broker_control_plane_listener_name` always point to a valid listener definition.